pkg/daemon/cleanup/cleanup.go

```diff
@@ -83,7 +83,7 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, configMapNamespace strin
 		return err
 	}
 	cleanupContainers()
-	return nil
+	return metadata.RemoveVersionAnnotation(ctx, directClient, *node)
```

Comment:
Its not clear why this is being done, please document what the full effects of the `Deconfigure`   function are in the function doc. All the high level changes to the node object and vm should be captured.